Download.Mvc/Client/components/Search.tsx

```diff
@@ -0,0 +1,51 @@
+﻿import * as React from 'react';
+import { match } from 'react-router-dom';
+import 'isomorphic-fetch';
+import { IProps } from './FolderView';
+import { SearchEntries } from './SearchEntries';
+import { Link } from 'react-router-dom';
+import { SearchBar } from './SearchBar';
+
+export class Search extends React.Component<IProps, any> {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            items: '',
+        };
+    }
+    componentDidMount = () => {
+        if (!this.props.match.params.query)
+            return;
+        fetch(`/api/Search/${decodeURIComponent(this.props.match.params.query)}`, { credentials: 'same-origin' })
```

Comment:
@laholstege curious if you need to decode the uri component since we are essentially just passing it back in a URL.  have you tested that this works w/ spaces or other characters in the search term?